package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "example-app-node-prisma",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
```

Comment:
I think `main` is for when you're importing the code as a module, but this is instead to be used as a binary so maybe it makes more sense to use [`bin`](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#bin)?